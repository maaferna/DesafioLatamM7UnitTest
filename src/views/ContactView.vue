<template>
    <div class="contact">
      <h1>Contact Us</h1>
      <p>Feel free to reach out to us with any questions!</p>
    </div>
  </template>
  
  <script>
  export default {
    name: 'ContactView',
  };
  </script>
  
  <style scoped>
  .contact {
    text-align: center;
    padding: 20px;
  }
  </style>
  