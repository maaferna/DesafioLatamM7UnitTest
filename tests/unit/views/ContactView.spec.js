import { mount } from '@vue/test-utils';
import ContactView from '@/views/ContactView.vue';

describe('ContactView', () => {
  test('renders the Contact view', () => {
    const wrapper = mount(ContactView);
    expect(wrapper.text()).toContain('Contact Us');
  });

  test('matches snapshot', () => {
    const wrapper = mount(ContactView);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
