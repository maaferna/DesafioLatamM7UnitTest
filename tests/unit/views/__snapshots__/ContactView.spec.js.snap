// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactView matches snapshot 1`] = `
<div class="contact">
  <h1>Contact Us</h1>
  <p>Feel free to reach out to us with any questions!</p>
</div>
`;
